salesforce_functions/_internal/cli.py

```diff
@@ -106,6 +104,13 @@ def check_function(project_path: Path) -> int:
 
 
 def start_server(project_path: Path, host: str, port: int, workers: int) -> int:
+    print(
+        f"Starting {PROGRAM_NAME} v{__version__}",
+        f"in multi-process mode ({workers} worker processes)."
+        if workers > 1
+        else "in single process mode.",
```

Comment:
I don't know Python super well, so just to double check: this does print `in multi-process [...]` only if the condition _after_ the string passes, otherwise the `single process` message?

Reply:
`print()` prints each of the provided arguments separated by a space (by default).

In Python, ternary expressions are written like `"foo" if True else "bar"`, so the second argument to `print()` will be the result of that expression.

The tests show the output for both branches of the conditional:
https://github.com/heroku/sf-functions-python/pull/16/files#diff-4e8715c7a425ee52e74b7df4d34efd32e8c92f3e60bd51bc2e1ad5943b82032eR186
https://github.com/heroku/sf-functions-python/pull/16/files#diff-4e8715c7a425ee52e74b7df4d34efd32e8c92f3e60bd51bc2e1ad5943b82032eR228

Reply:
I've refactored to make this implementation easier to read:
https://github.com/heroku/sf-functions-python/compare/29030c5f175cfb860e8f8b2091f7a22b3f670e01..5a106b58f5933f4d946f10a00a7aa0b1677e7e16